map/styled_point.cpp

```diff
@@ -0,0 +1,54 @@
+#include "map/styled_point.hpp"
+
+#include "base/stl_add.hpp"
+#include "base/string_utils.hpp"
+
+#include "std/fstream.hpp"
+#include "std/algorithm.hpp"
+#include "std/auto_ptr.hpp"
+
+
+graphics::DisplayList * StyledPoint::GetDisplayList(UserMarkDLCache * cache) const
+{
+  return cache->FindUserMark(UserMarkDLCache::Key(GetStyle(), graphics::EPosAbove, GetContainer()->GetDepth()));
```

Comment:
Sorry, my comment was a bit brief! What I mean is, my change means that the original ApiPoint mark is never drawn. One of the placemarks is always used instead. And the placemark drawing code is literally moved from the bookmark (KML) code, without significant change, so I think it ought to be right.

@mapsmetest How can I see the results of Jenkins tests?